src/server.ts

```diff
@@ -0,0 +1,9 @@
+export default (function server(){
+    const express = require('express')
+    const app = express()
+    const path = require("path")
+    app.use(express.static(path.join(__dirname,"public")))
+    app.use(express.json())
+    app.use(express.urlencoded({ extended: false }))
+    return app;
+})()
```

Comment:
Because of the way imports work, you don't have to do this as an IIFE; you can write this code exactly the same way without the surrounding function, and export the app result
```suggestion
import express from 'express';
import path from 'path';

const app = express();

app.use(express.static(path.join(__dirname, 'public')));
app.use(express.json());
app.use(express.urlencoded({ extended: false }));
export default app;
```